### DISCUSSION TITLE
thread/condition

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#12] [Announcements]
# thread/condition

一份通俗易懂、风趣幽默的Java学习指南，内容涵盖Java基础、Java并发编程、Java虚拟机、Java企业级开发、Java面试等核心知识点。学Java，就认准Java程序员进阶之路

https://tobebetterjavaer.com/thread/condition.html

**Comment by auqaoasis** [comment]
终于理解同步队列和等待队列了！

**Reply by itwanger** [reply]
可
